bin/run-bootstrap.sh

```diff
@@ -4,5 +4,7 @@ set -ex
 
 # install Node dependencies
 npm run development && npm run webpack:build
+# pre-render the svelte-based templates
+python manage.py compilejsi18n && npm run webpack:build:pre-render && python manage.py collectstatic --noinput
```

Comment:
Why do we need `compliejsi18n` and `collectstatic` here?

Reply:
even if we do need them, let's add the compilejsi18n first, then group together all the npm commands and leave the collectstatic to run after npm

Reply:
@akatsoulas Without running `compilejsi18n` prior, the `npm run webpack:build:pre-render` will fail, but perhaps we can remove the `collectstatic` step? I'll look into that.